### DISCUSSION TITLE
Personal daemon issue in IDz  during userbuild

### DISCUSSION THREAD

**Discussion by ratnamGT** [IBM/dbb-zappbuild#546] [Q&A]
Hi Team,

I am trying to run user build from the IDz and it is very slow so I am trying to use personal daemon . But I am getting bellow error while running below userbuild script from console

JVMJ9VM082E Unable to switch to IFA processor - issue "extattr +a libj9ifa29.so"

I am running below script 

$DBB_HOME/bin/groovyz -DBB_PERSONAL_DAEMON /u/gmsratn/zAppbuild20/Development/build.groovy --sourceDir /u/gmsratn/sandbox --workDir /u/gmsratn/sandbox/work --hlq GMSRATN --application MortgageApplication --userBuild 'MortgageApplication/cobol/hello.cbl'

It is not taking personal daemon and taking more than 5 minutes. Can anyone please help on this

**Comment by dennis-behm** [comment]
Hi @ratnamGT 

Thanks for reaching out. This is less a zAppBuild question, but rather a product related question. I encourage you to open a support request via the IBM support channels.

Unfortunately your question does not include which DBB version you are using. Please be aware, that DBB 2.0.2 introduces the [shared daemon](https://www.ibm.com/docs/en/dbb/2.0?topic=improving-performance-security-by-using-dbb-daemons). The personal daemon is deprecated and you should use the shared daemon going forward.
